rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -148,13 +179,16 @@ def _precompiled_apple_resource_bundle_impl(ctx):
             # the need to end up
             fail("Got unexpected target location '{}' for '{}'"
                 .format(target_location, sources_list))
-        input_files.extend(sources_list)
         for source in sources_list:
+            if not most_common_parent_resources_set.get(source):
+                continue
```

Comment:
will it be bad if we hit continue here? or it's simply not a resource `source` so we skip it?

Reply:
`most_common_parent_resources_set ` returns a Dict<Path, Boolean>. For the most common parent directories, the value is true, otherwise, it is false.  For example, if we have a `foo/` and `foo/bar` path, we don't want to include the `foo/bar` resources, so the dict will be 
```
foo/ -> True
foo/bar -> False
```
When this if statement hits `foo/bar`, we want to continue so this `foo/bar` is not included in the bundle.